phnt/include/ntldr.h

```diff
@@ -1075,86 +1266,101 @@ LdrNameOrIdFromResourceEntry(
         return (ULONG_PTR)Entry->Id;
 }
 
+// private
 NTSYSAPI
 NTSTATUS
 NTAPI
 LdrEnumResources(
     _In_ PVOID DllHandle,
-    _In_ PLDR_RESOURCE_INFO ResourceInfo,
-    _In_ ULONG Level,
-    _Inout_ ULONG *ResourceCount,
-    _Out_writes_to_opt_(*ResourceCount, *ResourceCount) PLDR_ENUM_RESOURCE_ENTRY Resources
+    _In_reads_(ResourceIdPathLength) PULONG_PTR ResourceIdPath, // PLDR_RESOURCE_ID
+    _In_ ULONG ResourceIdPathLength,
+    _Inout_ PULONG NumberOfResources,
+    _Out_writes_to_opt_(*NumberOfResources, *NumberOfResources) PLDR_ENUM_RESOURCE_ENTRY Resources
     );
 
+// private
 NTSYSAPI
 NTSTATUS
 NTAPI
 LdrFindEntryForAddress(
-    _In_ PVOID DllHandle,
-    _Out_ PLDR_DATA_TABLE_ENTRY *Entry
+    _In_ PVOID Address,
```

Comment:
Address should be BaseAddress since it's the same as NtQueryVirtualMemeory: https://learn.microsoft.com/en-us/windows-hardware/drivers/ddi/ntifs/nf-ntifs-zwqueryvirtualmemory

Reply:
The address doesn't have to be the base address though; it could be anything within the DLL limits